docs/source/getting_started/authoring_your_workflow.md

```diff
@@ -18,91 +18,85 @@ The tutorial aims to be an extension to the quickstart to help you better unders
+│   ├── bowtie2
+│   └── ...
 ├── reference
 │   ├── wuhan.1.bt2
-│   ├── wuhan.2.bt2
-│   ├── wuhan.3.bt2
-│   ├── wuhan.4.bt2
-│   ├── wuhan.fasta
-│   ├── wuhan.rev.1.bt2
-│   └── wuhan.rev.2.bt2
+│   └── ...
+├── system-requirements.txt
 ├── version
 └── wf
-    └── __init__.py
+    ├── __init__.py
+    ├── assemble.py
+    └── sort.py
 ```
+
 Once your boilerplate workflow has been created successfully, you should see a folder called `covid-wf`.
 
 ## 2. Writing your First Task
 
-A task is a pure Python function that takes in inputs and returns outputs. Here, we are writing a task that takes in two sequencing reads and returns an assembled SAM file.
+A task is a Python function that takes in inputs and returns outputs to the Latch platform or to another task. In the `assemble.py` file inside the `covid-wf/wf` directory, we have a task which ingests two sequencing reads and outputs an assembled SAM file.
+
+### How to work with LatchFiles and LatchDirs
+
+`LatchFile` and `LatchDir` are types built into the Latch SDK which allow users to use files and directories on Latch as inputs and outputs to workflows. They point to remote file locations in our user interface on [Latch Console](https://console.latch.bio/data) and implement the necessary operations to ensure data is pulled into the task environment and saved back into your data tab.
+
+Any time you read or resolve the path of a file or directory in a task, the data will be downloaded locally and will be accessible within the task. For example the line `local_file = Path(read1)` in the snippet below will download the file from Latch and return a path to the local file.
+
 
 ```python
+from latch.types import LatchFile, LatchOutputDir
+...
+
 @small_task
-def assembly_task(read1: LatchFile, read2: LatchFile) -> LatchFile:
-
-    # A reference to our output.
-    sam_file = Path("covid_assembly.sam").resolve()
-
-    _bowtie2_cmd = [
-        "bowtie2/bowtie2",
-        "--local",
-        "-x",
-        "wuhan",
-        "-1",
-        read1.local_path,
-        "-2",
-        read2.local_path,
-        "--very-sensitive-local",
-        "-S",
-        str(sam_file),
-    ]
-
-    subprocess.run(_bowtie2_cmd)
-
-    return LatchFile(str(sam_file), "latch:///covid_assembly.sam")
-```
+def assembly_task(
+    read1: LatchFile, read2: LatchFile, output_directory: LatchOutputDir
+) -> LatchFile:
 
-### How to work with LatchFiles
+    local_file = Path(read1)
+    ...
+```
 
-`LatchFile` refers to a remote file location on
-[Latch Console](https://console.latch.bio/data).
+Sometimes, a python task does not directly access a file/directory, but uses it implicitly; for example, through a subprocessed shell command. We can ensure that the file's data is present by either calling `Path` on the file or directory, or by accessing the `local_path` attribute of the LatchFile or LatchDir. This will download the data into the task environment and return back a path to the local file or directory.
```

Comment:
"but uses it implicitly" -> "but uses its path"
instead of ";", split into two sentences

"We can ensure that the file's data is present by either calling `Path` on the file or directory, or by accessing the `local_path` attribute of the LatchFile or LatchDir...." weird, especially given that the last paragraph already explained how to download shit